### ISSUE TITLE
[Accessibility] Background color of configuration screens inaccessible

### ISSUE THREAD

**Issue by glenn-sorrentino** [guardianproject/haven#227]
The yellow text color (#F5C229) doesn’t have enough contrast against the blue background color (#2196F3) to make the app unusable for some users with color blindness. It can cause strain on the eyes of users with good vision, and block users with poor vision entirely.

If you want to keep the current yellow text, darken the blue to #085696 to make the contrast pass (Option 1 below). But there's also an opportunity to align all of the dark blue background colors and font-colors an all of the onboarding & configuration screens (Option 2).

You can check colors to see if they pass here: https://webaim.org/resources/contrastchecker/

Note: The explorations show more margin around the UI text.

Current Configuration Screen:
![configuration - current](https://user-images.githubusercontent.com/28545431/35119067-a5f8cf18-fc48-11e7-9dfe-93ef0eb51871.png)

Accessible Colors - Option 1:
![configuration - accessible colors](https://user-images.githubusercontent.com/28545431/35118985-776f1350-fc48-11e7-8003-42ffdfef9280.png)

Accessible Colors - Option 2:
![configuration - consistent accessible colors](https://user-images.githubusercontent.com/28545431/35119027-8b86de18-fc48-11e7-9b64-38b0f2d32ffb.png)

Pages from the [Design Audit ](https://github.com/guardianproject/haven/issues/218)

[Configuration.pdf](https://github.com/guardianproject/haven/files/1644327/Configuration.pdf)
![configure - accessibility](https://user-images.githubusercontent.com/28545431/35119486-f08e4278-fc49-11e7-8ab8-a7e312e80a19.jpg)

[Configuration - Current Colors.pdf](https://github.com/guardianproject/haven/files/1644330/Configuration.-.Current.pdf)
![configure - current colors](https://user-images.githubusercontent.com/28545431/35119587-4afbde3c-fc4a-11e7-9155-3d6d8d25eb7b.jpg)

[Configuration - Accessible Colors.pdf](https://github.com/guardianproject/haven/files/1644326/Configuration.-.Accessible.Colors.pdf)
![configure - accessible colors](https://user-images.githubusercontent.com/28545431/35119595-5543aba4-fc4a-11e7-9163-2a267435ee66.jpg)

**Comment by n8fr8** [comment]
Thanks @fat-tire will take a look at the PR.
